app/Widgets/Hero_Slider_Widget.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Boilerplate\Theme\Widgets;
+
+use JustCoded\WP\Framework\Page_Builder\v25\Page_Builder_Widget;
+use JustCoded\WP\Framework\Web\View;
+
+/**
+ * Class Hero_Slider_Widget
+ * Custom image wordpress widget based on siteorigin widgets bundle
+ */
+class Hero_Slider_Widget extends Page_Builder_Widget {
+
+	/**
+	 * Default slider type
+	 */
+	const TYPE_FULL = 'hero-slider-full';
+
+	/**
+	 * Hero_Slider_Widget constructor.
+	 */
+	function __construct() {
+		parent::__construct(
+			'hero-slider',
+			__( 'Slider image widget' ),
+			array(
+				'description' => __( 'Slider image widget.' ),
+				'has_preview' => false,
+			),
+			array(),
+			false,
+			''
+		);
+	}
+
+	/**
+	 * Return a list of images to be shown as preview.
+	 * By default it's widget {id_base}.png with $this->name
+	 *
+	 * @return array
+	 */
+	public function get_preview_images() {
+		return array(
+			'hero-slider.png' => 'Preview of the slider widget',
+		);
+	}
+
+	/**
+	 * Form fields configuration
+	 *
+	 * @return array
+	 */
+	function get_widget_form() {
+		return
+			array(
+				'images'      => array(
+					'type'       => 'repeater',
+					'label'      => __( 'Images', 'wordpress-theme-boilerplate' ),
+					'item_name'  => __( 'Image', 'wordpress-theme-boilerplate' ),
```

Comment:
we don't have such textdomain. check other files for correct textdomain

Reply:
renamed